packages/fluentui/react-northstar/src/components/Accordion/Accordion.tsx

```diff
@@ -276,17 +276,19 @@ export const Accordion = (React.forwardRef<HTMLDListElement, AccordionProps>((pr
           render: renderPanelTitle,
         }),
       );
-      children.push(
-        createShorthand(AccordionContent, content, {
-          defaultProps: () => ({
-            className: accordionSlotClassNames.content,
-            active,
-            id: contentId,
-            accordionTitleId: titleId,
+      if (active) {
```

Comment:
IMO we should add a test for this with enzyme/testing-library to assert that this DOM no longer appears in a snapshot